static/script.js

```diff
@@ -69,3 +70,26 @@ function enableAutoRefresh () { // eslint-disable-line no-unused-vars
     }
   }
 }
+
+function unfoldAll () { // eslint-disable-line no-unused-vars
+  window.setInterval(unfoldButton, 100)
+  $('#unfold-all-li').removeClass('d-none')
+  let allUnfoldIds = $('#refs_list').attr('content').replace(/[\[\]'"\ ]/gm, '').split(',').sort().toString() // eslint-disable-line
+
+  $('#unfold-all').on('click', function (event) {
+    if (document.getElementById('unfold-all').checked === false) {
```

Comment:
Be consistent. If you are using jQuery for events and handlers, don't use vanilla document methods.
You can change it to:
```javascript
if (! $(this).checked) {
  // do stuff
}
```
Attribute `checked` will return `true` or `false`. You don't have to explicitly compare with `=== false`